pkgpanda/util.py

```diff
@@ -62,7 +63,16 @@ def remove_directory(path):
         # so calling out to the cmd prompt to do this fixes that.
         path = path.replace('/', '\\')
         if os.path.exists(path):
-            subprocess.call(['cmd.exe', '/c', 'rmdir', '/s', '/q', path])
+            for retry in range(1, 10):
```

Comment:
Although the implementation appears to be correct, I would prefer to see this done using one of the Python `retry` or `retrying` modules.